.pre-commit-config.yaml

```diff
@@ -0,0 +1,42 @@
+# Copyright (c) 2019 Shotgun Software Inc.
```

Comment:
It was necessary to add this file now since our Azure build pipeline expects to be able to run pre-commit.